### ISSUE TITLE
Understanding the reason for an instruction

### ISSUE THREAD

**Issue by Temidayo32** [moja-global/FLINT#110]
**What does the current documentation state?**
![Screenshot_20211018-081338](https://user-images.githubusercontent.com/87539866/137817952-1c8e8c47-97f2-4619-b649-1d7436290f6b.png)

**Where is this stated?**
[This is a link to the page where the issue is raised from](https://docs.moja.global/en/latest/DevelopmentSetup/git_and_github_guide.html) 

**Why do you want to improve the statement?**
The need to improve the circled section of the page hinges upon a need to better understand why the instruction required of the user of the doc was given. That is, answering the question on, "why the user should make sure to have checked out to the develop Branch for the FLINT Installation?"
What is the reason for this requirement here? What is the implications of not taking the required action? 
I strongly believe this is a knowledge gap that needs to be filled in to create better documentation and satisfy users' innate need to understand why they have to take a step.

**Proposed statement**
The proposed solution, hence, is to state clearly the reason for the action requirement, alongside possible implication of not taking the step.

**Comment by Mohitkumar6122** [comment]
@Knowledgeissweet, that's a good catch!.
Since the ```develop``` branch is the branch where all the latest technical changes are merged into ( code enhancement, adding new features or removing dead code etc) by the developers. So in terms of the latest branch (in terms of technical aspect), the ```develop``` branch is the best to work with.
Also, the modules are all written to work with the ```develop``` branch of the FLINT and any actively maintained branches will work with the most recent version (and are broken with old enough builds of the FLINT). 

Maybe we can include it in the documentation.

Tagging @chicken-biryani, @HarshCasper, @shubhamkarande13 for more insight into this!

**Comment by shloka-gupta** [comment]
@Knowledgeissweet that's a great suggestion.

Maybe one or two lines would be enough.

Something like:

'Make sure you've switched to the develop branch since it consists of all the latest changes.'

Or something similar.

**Comment by King-kay** [comment]
@HarshCasper , something I can do??

**Comment by King-kay** [comment]
" Make sure you've switched to the develop branch To be updated, it consists of all the latest changes." @chicken-biryani does this suffice?

**Comment by shloka-gupta** [comment]
@King-kay yep. Sounds reasonable!

**Comment by King-kay** [comment]
https://github.com/moja-global/moja_global_docs/pull/144
